foo.js

```diff
@@ -0,0 +1,28 @@
+import Alt, { Store } from './'
+
+const alt = new Alt()
+
+const actions = alt.generateActions('foo')
+
+class MyStore extends Store {
+  constructor() {
+    super(alt, {
```

Comment:
If you already know that this is a `Store` (from the base class), maybe something like an `@app.register` decorator to connect it to the app and return a singleton? Basically the same thing as `alt.createStore`, but without repeating the "store" part in the name.